packages/dds/cell/src/test/cell.spec.ts

```diff
@@ -308,35 +311,50 @@ describe("Cell", () => {
             private readonly cell2: ISharedCell;
             private readonly containerRuntimeFactory: MockContainerRuntimeFactory;
 
-            constructor() {
+            public constructor() {
                 this.containerRuntimeFactory = new MockContainerRuntimeFactory();
                 this.cell1 = createConnectedCell("cell1", this.containerRuntimeFactory);
                 this.cell2 = createConnectedCell("cell2", this.containerRuntimeFactory);
             }
 
-            public get sharedObject() {
+            /**
+             * {@inheritDoc @fluid-internal/test-dds-utils#IGCTestProvider.sharedObject}
+             */
```

Comment:
I could see some pushback against requiring documentation for all public members in classes in test files, but I think it's worth it in the end.

Reply:
That's a fair point. We can always omit test code from those rules if we want to, but I also think tests (can) serve as great usage documentation, so making sure test code is thoroughly documented can definitely provide value.